server/backend/database/client_info.go

```diff
@@ -82,7 +82,7 @@ type ClientInfo struct {
 func (i ClientInfo) CheckIfInProject(projectID types.ID) error {
 	if i.ProjectID != projectID {
 		return fmt.Errorf(
-			"client not in project: clientID: %s, client's projectID: %s, target projectID: %s, %w",
+			"check client(%s,%s) in project(%s): %w",
```

Comment:
Change the error string differently for each method
- `CheckIfInProject()`: `check client(%s,%s) in project(%s): %w`
- `AttachDocument()`: `client(%s) attaches document(%s): %w`
- `IsAttached()`: `check document(%s) is attached: %w`
- `UpdateCheckpoint()`: `update checkpoint in document(%s): %w`
- `EnsureDocumentAttached()`: `ensure attached document(%s) in client(%s): %w`